### DISCUSSION TITLE
Can't get the WebRTC stream running

### DISCUSSION THREAD

**Discussion by alenzurfluh** [blakeblackshear/frigate#9245] [Ask A Question]
Hi, 
I'm using  Frigate with go2rtc in a docker container. I've updated go2rtc to the latest version because I want to use ISAPI for 2-way audio for my Hikvision doorbell camera.

I've have defined following stream for my camera.

```
  streams:
     haustuere_rtps:
      - "rtsp://admin:XXX@CAM_HAUSTUERE:554/Streaming/Channels/101/"
      - "isapi://admin:XXX@CAM_HAUSTUERE:80"
```

In Frigate under Live view my camera is working when I select mse or jsmeg. But the video is not loading in the webrtc mode. But I need that for 2 way-audio.

The only thing I can see in go2rtc log is:

```
2024-01-07 17:31:31.683296165  17:31:31.683 DBG [streams] stop producer url=isapi://admin:XXXXXXXX@CAM_HAUSTUERE:80
2024-01-07 17:31:32.195728449  17:31:32.195 DBG [streams] can't stop none producer
```
what does ` can't stop none producer` mean?

I've mapped port 8555 from the docker container to my host.

![image](https://github.com/blakeblackshear/frigate/assets/37574827/d3bb643c-fe9c-4752-b005-156cd1c23f22)

I've also tried to open the WebRTC link directly from the go2rtc dashboard. But that's also not working. Just getting a black screen with the video not loading.

![image](https://github.com/blakeblackshear/frigate/assets/37574827/c4330729-0401-484e-a1a0-af11aa0e9dc9)

What else am I missing to WebRTC working?

Not sure about this section in the documentation. Given I'm running Frigate/go2rtc in a docker do I need to add the internal ip address of the that docker container under candidates in the config file?

![image](https://github.com/blakeblackshear/frigate/assets/37574827/c9517e2e-86d3-4331-97f1-fc4e5d9f051f)

Thanks

**Comment by NickM-27** [comment]
What is the stream info for the hikvision? If I had to guess it might be h265

**Reply by alenzurfluh** [reply]
according VLC and ffprobe the stream is h264
![image](https://github.com/blakeblackshear/frigate/assets/37574827/93df81a8-815b-4cfb-946b-2c75c9e62ad0)


I've now changed the network mode from bridge to host and WebRTC is working. So it must have something to do with the network/ports

**Comment by HIhnen** [comment]
Try another Browser. I never got WebRTC running with Firefox, with Chrome it worked instantly.
Also the camera streams are starting faster with Chrome.
